code/game/machinery/doors/airlock.dm

```diff
@@ -1311,6 +1318,13 @@ About the new airlock wires panel:
 		// If we lost power, disable electrification
 		// Keeping door lights on, runs on internal battery or something.
 		electrified_until = 0
+		//if we lost power open 'er up
+		if(insecure)
+			open(1)

```

Comment:
Could also `unlock(1)` so it unbolts (by force.)

Reply:
That is not desired behavior.